build/push-image.sh

```diff
@@ -44,3 +44,16 @@ docker push cortexlabs/${image}:${CORTEX_VERSION}
 if [ "$slim" == "true" ]; then
```

Comment:
As discussed, we decided to not push e.g. `cortexlabs/python-predictor-gpu-slim:<version>`